cmd/ydbcp/main.go

```diff
@@ -119,7 +119,7 @@ func (s *server) MakeBackup(ctx context.Context, req *pb.MakeBackupRequest) (*pb
 		SourcePaths:         req.GetSourcePaths(),
 		SourcePathToExclude: req.GetSourcePathsToExclude(),
 		DestinationPrefix:   s.s3.PathPrefix,
-		BackupID:            types.GenerateObjectID(), // TODO: do we need backup id?
+		BackupID:            types.GenerateObjectId(), // TODO: do we need backup id?
```

Comment:
Better to use ID instead of Id as recommended in golang styleguide
https://google.github.io/styleguide/go/decisions#initialisms